### PR TITLE
Add option for semileptonic decay of Omegac0

### PR DESCRIPTION
**Pull request by chengtt0406**
Add option to force Omegac0 -> Omega+e+nu_e decay, which is needed for D2H MC production

### CODE REVIEW

File: PYTHIA8/AliPythia8/AliDecayerPythia8.h [hunk 1]
```diff
@@ -39,15 +39,15 @@ class AliDecayerPythia8 : public TVirtualMCDecayer {
...
+  void     ForceHadronicD(Int_t optUser4Bodies = 1,Int_t optUseDtoV0 =1, Int_t optForceLcChannel = 0, Int_t optForceXicChannel = 0, Int_t optForceDsChannel = 0, Int_t optForceOmegacChannel = 0);
   void  ForceBeautyUpgrade();
   AliTPythia8*  fPythia8;          // Pointer to pythia8
   Int_t         fDebug;            // Debug level
  
   Decay_t       fDecay;            //  Forced decay mode
   Bool_t        fHeavyFlavour;     //! Flag for heavy flavors
   static Bool_t fgInit;            //! initialization flag 
-  ClassDef(AliDecayerPythia8, 2) // Particle Decayer using Pythia8
+  ClassDef(AliDecayerPythia8, 3) // Particle Decayer using Pythia8
```
Comment (chiarazampolli) [inline PYTHIA8/AliPythia8/AliDecayerPythia8.h:50]: Why did you increase the ClassDef? You did not add any data member, unless I am overlooking something.

Comment (chengtt0406) [reply PYTHIA8/AliPythia8/AliDecayerPythia8.h:50]: You are right, I did not add new data member, just modified something.  Sorry, I do not know the rules in detail.  I will keep '2', and recommit this file again soon. Thanks a lot.

Comment (chengtt0406) [reply PYTHIA8/AliPythia8/AliDecayerPythia8.h]: @chiarazampolli Could you please have a look about the new PRs and approve it. Thank you very much

**Comment by chiarazampolli** [comment]
Seems fine to me. 
+1

**Comment by gconesab** [comment]
Hi, This commit is still not merged, @chiarazampolli , I think you should start the comment with +1 :)

**Comment by chengtt0406** [comment]
@chiarazampolli  Could you please approve this commit? Thanks a lot :)

**Comment by chengtt0406** [comment]
@gconesab  Hi, can you please approve this commit if it is ok for you? Thank you very much :))

**Comment by gconesab** [comment]
> @gconesab Hi, can you please approve this commit if it is ok for you? Thank you very much :))

Unfortunately, I do not have the powers to do that.

**Comment by chiarazampolli** [comment]
+1

**Comment by gconesab** [comment]
Hi @shahor02, @maireiphc  and others, can any of you approve? Thanks

**Comment by shahor02** [comment]
+1

**Pull request merged**